### ISSUE TITLE
The new device contribution guide needs to be updated

### ISSUE THREAD

**Issue by floion** [resin-os/resinos#262]
The guide https://github.com/resin-os/meta-resin/blob/master/contributing-device-support.md must be updated to reflect latest changes in meta-resin 2.0.0

**Comment by floion** [comment]
Any progress on this?

**Comment by agherzan** [comment]
@floion there was nothing done in this regards hence assigning back to you.

**Comment by floion** [comment]
I was under the impression that the OS team should handle this since it is closely related to meta-resin.

**Comment by alexandrosm** [comment]
@floion arguably, this is about device support, so it's somewhere in the
middle..

-- 

*Alexandros Marinos*

Founder & CEO, Resin.io


+1 206-637-5498

@alexandrosm

**Comment by floion** [comment]
yes, it may be in the middle, but it sits in the meta-resin repository and it is actually describing features from meta-resin
